gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -147,3 +159,79 @@ def expand(self, pcoll):
                       beam.io.BigQueryDisposition.WRITE_APPEND
                       if self._append
                       else beam.io.BigQueryDisposition.WRITE_TRUNCATE))))
+
+  def _update_bigquery_schema(self, update_schema):
+    # type: (bool) -> None
+    if not update_schema or not self._schema:
+      return
+    # if table does not exist, do not need to update the schema.
+    output_table_re_match = re.match(
```

Comment:
Ideally, this should be combined with the logic in validate(), but please just add a TODO for now :)